settings/request_test.go

```diff
@@ -76,7 +72,10 @@ func TestRequestsAPI(t *testing.T) {
 
 		var reqBuf bytes.Buffer
 		enc := msgpack.NewEncoder(&reqBuf)
-		require.Nilf(t, test.req.Body(&resolver, enc), "No errors on fill")
+		require.Nilf(t, test.req.Body(&resolverNames, enc),
+			"No errors on fill, names supported")
+		require.Nilf(t, test.req.Body(&resolverNoNames, enc),
+			"No errors on fill, names unsupported")
```

Comment:
It looks like a two different tests or tests cases. Anyway, what the test actually test?

Reply:
Updated. This test tests equests API, so I removed use of `NamesUseSupported()`.